File: repos/beecrowd/javascript/bee1020.js
let dias = parseInt(require("fs").readFileSync("stdin", "utf8"));

const anos = dias / 365;
dias %= 365;
const meses = dias / 30;
dias %= 30;

console.log(`${parseInt(anos)} ano(s)`);
console.log(`${parseInt(meses)} mes(es)`);
console.log(`${parseInt(dias)} dia(s)`);

File: c/bee1020.c
#include <stdio.h>
 
int main() {
    int anos, meses, dias;
    scanf("%d", &dias);

    anos = dias/365;
    dias %= 365;
    meses = dias/30;
    dias %= 30;

    printf("%d ano(s)\n", anos);
    printf("%d mes(es)\n", meses);
    printf("%d dia(s)\n", dias);
    return 0;
}